jobs/gtfs-rt-parser/test_gtfs_rt_parser.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
```

Comment:
**question:** How is this test intended to be used? (Run manually by developer? Are we adding any automated testing anywhere that I'm not understanding?)

Reply:
This is executed via `pytest` and is mostly for developer usage. It's tricky since we have multiple venvs/requirements, but running tests per Docker image is maybe a good idea.

Reply:
Yes I will add a README.

In more mature CI/CD pipelines, the order of operations is usually build image => run tests => deploy image, so that should be our north star IMO.